Sources/SmithyHTTPAuth/SigV4Signer.swift

```diff
@@ -52,7 +52,13 @@ public class SigV4Signer: SmithyHTTPAuthAPI.Signer, @unchecked Sendable {
             )
         }
 
-        let signingConfig = try constructSigningConfig(identity: identity, signingProperties: signingProperties)
+        let signedAt = Date()
```

Comment:
The signer gets current `Date` and keeps it in a var, so the same date can be used for signing and for updating the `HTTPRequest.signedAt` property.